Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} objective of this project is create a cold gas thruster. The CGT will allow the team to gain experience in designing, building, and testing propulsion systems.
+There will be two types of nozzles tested, a minimum length nozzle and an aerospike nozzle. The nozzles will be optimized for ambient conditions.
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster capable of sustained cold burns which can be applied to a CubeSat to achieve the reuqired
+velocity to reenter Earth's atmosphere. This is a system property, however using the thrust over time curves reasonable estimations can be made to say whether or not it is possible.
+A target specific impulse is 50s. This can be determined from testing by integrating the thrust over time curve. See the equation 1, below, to see why this is possible.
+
+\begin{equation} \label{specific impulse}
+  I_t    = \int_{t_i}^{t_f} F_{thrust} dt
+\end{equation}
+
+\begin{equation} \label{specific impulse}
+  I_s = \frac{I_t}{m_p g_0}
+\end{equation}
+
+Where $m_p$ is the propellant mass and $g_0$ is standard gravity, 9.81 $ms^{-2}$.
+As seen above, calculating the specific impulse is
+
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/cold-gas-thruster.png}
+  \caption{This is the Solidworks Assembly of the current design of the SPEX cold gas thruster. The piping interface to the nozzle is omitted for clarity.}
+\label{fig:lifecycle}
+\end{figure}
+
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data.
+However, from previous testing noise is not a problem but error is slightly. The error in the last tests maxed at 0.05 Newtons, which is no terribly significant but still should be accounted.
+Vacuum condition availablity is low at RIT, so the team will need to account for ambient conditions. This primarily includes ambient pressure for achieveing maximum thrust, but there are other
+nuisances. The use of this cold gas thruster will give the team experience in the fabrication of propulsion systems and that can be used for designing more complex systems. These systems could
```

Comment:
Agree. When writing technical documents, try your best to speak in objective terms.